src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -428,7 +431,9 @@ async Task<T> Execute<T>(RestRequestWithCache request, Func<IRestResponse, T> se
                     break;
 
                 case RestRequestWithCache.CacheStyle.IfNotModified:
+#pragma warning disable IDE0010 // Add missing cases
```

Comment:
Too many to add?

Reply:
Yeah, every HTTP status code is in that enum.